Makefile

```diff
@@ -136,6 +136,13 @@ $(BINDIR)/lister-gen:
 	  --extra-peer-dirs k8s.io/kubernetes/pkg/api,k8s.io/kubernetes/pkg/api/v1,k8s.io/kubernetes/pkg/apis/meta/v1,k8s.io/kubernetes/pkg/conversion,k8s.io/kubernetes/pkg/runtime \
 	  -O zz_generated.conversion
 	# the previous three directories will be changed from kubernetes to apimachinery in the future
+	# Generate the internal clientset (pkg/client/clientset_generated/internalclientset)
+	$(DOCKER_CMD) $(BINDIR)/client-gen \
+		--input-base "github.com/kubernetes-incubator/service-catalog/pkg/apis/" \
+		--input servicecatalog/ \
```

Comment:
Should we feed the knowledge about the trailing slash into the docs? (maybe you've done this already)